extensions/terminal-suggest/src/completions/code-tunnel.ts

```diff
@@ -0,0 +1,20 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+import code, { commonOptions, extensionManagementOptions, troubleshootingOptions, tunnelOptions, tunnelTopLevelSubcommands } from './code';
+
+const codeTunnelCompletionSpec: Fig.Spec = {
+	...code,
+	name: 'code-tunnel',
+	description: 'Create a tunnel that\'s accessible on vscode.dev from anywhere.',
+	subcommands: tunnelTopLevelSubcommands,
```

Comment:
I don't think we support the full list of subcommands yet?

![image](https://github.com/user-attachments/assets/fddc3f22-c592-452b-884d-733b2e2877c5)

We need to reopen https://github.com/microsoft/vscode/pull/243809 if so